src/game/Render/Render.cpp

```diff
@@ -77,3 +77,31 @@ bool R_PushRemoteScreenUpdate(std::int32_t a1)
     }
     return IsMainThread;
 }
+
+//DONE : 0x004CFA00
+std::int32_t FixWindowsDesktop()
+{
+    unsigned long currentThreadId = GetCurrentThreadId();
```

Comment:
this should be `DWORD`, not `unsigned long`

Reply:
DWORD is a macro, they mean the same thing, we are trying (keyword is "trying") to maintain compatibility across tool sets.
![image](https://user-images.githubusercontent.com/10711154/166823000-356d84cb-a9ea-452d-8af7-1b5be63fc4f5.png)